projects/isy-angular-widgets-demo/src/app/feature/objekt-anzeigen/objekt-anzeigen.component.html

```diff
@@ -1,301 +1,352 @@
   [toggleable]="true"
 >
-  <div class="grid m-0">
-    <div class="col-1 text-right align-self-center">
-      <label for="basis-nachname">
-        {{ 'isyAngularWidgetsDemo.labels.nachname' | translate }}
-      </label>
-    </div>
-    <div class="col">
-      <input
-        id="basis-nachname"
-        class="w-full"
-        type="text"
-        pInputText
-        disabled
-        [(ngModel)]="person.personalien.nachname"
-      />
-    </div>
-    <div class="col-1 text-right align-self-center">
-      <label for="basis-vorname">
-        {{ 'isyAngularWidgetsDemo.labels.vorname' | translate }}
-      </label>
-    </div>
-    <div class="col">
-      <input
-        id="basis-vorname"
-        class="w-full"
-        type="text"
-        pInputText
-        disabled
-        [(ngModel)]="person.personalien.vorname"
-      />
-    </div>
-    <div class="col-1 text-right align-self-center">
-      <label for="basis-geburtsdatum">
-        {{ 'isyAngularWidgetsDemo.labels.geburtsdatum' | translate }}
-      </label>
-    </div>
-    <div class="col">
-      <input
-        id="basis-geburtsdatum"
-        class="w-full"
-        type="text"
-        pInputText
-        disabled
-        [(ngModel)]="person.personalien.geburtsdatum"
-      />
-    </div>
-  </div>
-</p-panel>
-
-<p-tabView id="tab-view">
-  <p-tabPanel id="panel-personalien" [header]="'isyAngularWidgetsDemo.displayObject.tabPersonalData' | translate">
-    <form [formGroup]="personalInfoForm">
-      <div class="grid m-0">
-        <div class="col-1 text-right align-self-center">
-          <label for="last-name">
-            {{ 'isyAngularWidgetsDemo.labels.nachname' | translate }}
-          </label>
-        </div>
-        <div class="col">
-          <input id="last-name" formControlName="lastName" type="text" pInputText isyInputChar />
-          <small
-            class="p-error block"
-            *ngIf="personalInfoForm.controls['lastName'].invalid && personalInfoForm.controls['lastName'].touched"
-          >
-            {{ 'isyAngularWidgetsDemo.messages.requiredMessage' | translate }}
-          </small>
-        </div>
-        <div class="col-1 text-right align-self-center">
-          <label class="align-content-end" for="birth-name">
-            {{ 'isyAngularWidgetsDemo.labels.geburtsname' | translate }}
-          </label>
-        </div>
-        <div class="col">
-          <input id="birth-name" formControlName="birthName" type="text" pInputText isyInputChar />
-        </div>
-        <div class="col-1 text-right align-self-center">
-          <label class="align-content-end" for="birth-place">
-            {{ 'isyAngularWidgetsDemo.labels.geburtsort' | translate }}
-          </label>
-        </div>
-        <div class="col">
-          <input id="birth-place" formControlName="birthplace" class="w-full" type="text" pInputText />
+  <div class="grid mt-1 align-items-center">
```

Comment:
align-items-center is necessary because otherwise the input fields are no longer aligned horizontally correctly when using long labels that extend over several lines.
![Screenshot 2024-02-08 182930](https://github.com/IsyFact/isy-angular-widgets/assets/74294456/ccb25d66-ebb1-49fe-9c34-b45cd178fbeb)